### DISCUSSION TITLE
点配置没反应也不弹出登录怎么办？

### DISCUSSION THREAD

**Discussion by Fc8cF** [mikumifa/biliTickerBuy#244] [提问专区]
在这个页面点了配置没反应，听别人说可能要等一会儿会弹出登录，但是我这边没有
![4](https://github.com/mikumifa/biliTickerBuy/assets/146006959/addd28a0-85f6-48c5-8ab5-c669df8697c7)
然后就想手动导入试试看，但是导入的话就一直在转圈圈，也登录不上，这是怎么回事呀？
梯子开着关了都是这样😭
![Uploading 3.png…]()

**Comment by mikumifa** [comment]
没登录

**Reply by Fc8cF** [reply]
请问要怎么登录呀？

**Reply by Fc8cF** [reply]
我用cookies.json导入也登录不上，也没有登录页面弹出TT
